src/app/content/highlights/hooks/loadMore.ts

```diff
@@ -56,10 +54,13 @@ export const hookBody: ActionHookBody<
     try {
       highlights = await loadMore(services, summaryPageSize);
     } catch (error) {
-      Sentry.captureException(error);
-      services.dispatch(addToast(toastMessageKeys.higlights.failure.popUp.load, {destination: 'myHighlights'}));
       services.dispatch(toggleSummaryHighlightsLoading(false));
-      return;
+
+      if (error instanceof CustomApplicationError) {
```

Comment:
you're doing this if check a lot, i feel like we could do something to streamline it like:

```ts
// not loving the name, maybe something better
throw CustomApplicationError::check(error, new HighlightPopupLoadError({ destination: 'myHighlights' }));
```

```ts
export class CustomApplicationError extends Error {
  public static function check(error: any, customError: CustomApplicationError) {
    if (error instanceof CustomApplicationError) {
      return error;
    }
    return customError;
  }
}
```

or something